ahmed-yara-sara/src/components/Footer.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { fa } from "@fortawesome/fontawesome-free";
+import {
+  faFacebookSquare,
+  faTwitterSquare,
+  faInstagramSquare,
+  faGithubSquare,
+} from "@fortawesome/free-brands-svg-icons";
+
+class FooterBar extends Component {
+  render() {
+    return (
+      <footer className="pt-4 my-md-5 pt-md-5 border-top bg-dark page-footer">
+        <div className="text-center text-light">
+          {" "}
+          <h3>This Website was made by</h3>{" "}
```

Comment:
remove this random empty strings here and every where in your code
```suggestion
          <h3>This Website was made by</h3>
```